test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -1141,3 +892,60 @@ type allResourceSource struct{}
 
 func (*allResourceSource) AnyVersionForGroupEnabled(group string) bool     { return true }
 func (*allResourceSource) VersionEnabled(version schema.GroupVersion) bool { return true }
+
+type resourceToPersist struct {
+	gvk        schema.GroupVersionKind
+	gvr        schema.GroupVersionResource
+	golangType reflect.Type
+	namespaced bool
+	isOAPI     bool
```

Comment:
Sorry I missed this on my first pass review.

Reply:
> Sorry I missed this on my first pass review.

fixed